R/calcCapacity.R

```diff
@@ -72,12 +72,19 @@ calcCapacity <- function(subtype) {
     # renaming technologies to REMIND naming convention
     WEOcap <- rename_dimnames(WEOcap, dim = 3, query = mapping, from = "WEO_techs", to="REMIND_techs")
     WEOcap <- WEOcap * 1E-03 # converting GW to TW
-
+    
+#*** near term China gas data - see below
```

Comment:
maybe add a comment: "creating empty lines to be filled by hand below"?

Reply:
> I assume you checked the output and it contains the right values :-)

yes, I did ^^

Reply:
> maybe add a comment: "creating empty lines to be filled by hand below"?

Thanks, this works